lib/drives/models/recurring_drive.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:rrule/rrule.dart';
+
+import '../../account/models/profile.dart';
+import '../../util/parse_helper.dart';
+import '../../util/search/position.dart';
+import '../../util/supabase_manager.dart';
+import '../../util/trip/trip_like.dart';
+import 'drive.dart';
+
+class RecurringDrive extends TripLike {
+  DateTime? stoppedAt;
```

Comment:
Why do we need the information when it was stopped and not only that is was stopped?

Reply:
I thought we might add the option to say "Stop next week"/"Stop at specific date" or something like that? Otherwise there is no way to stop the recurring drive without having a drive scheduled literally the next day, where people might already ride with you.

And if not, this is still good practice because it stores interesting information for us.

Reply:
Okay but you didn't actually implement that in the Postgres functions correctly I think

Reply:
I think I did that correctly live? But yeah you are right, and I will double-check